src/main/java/bio/terra/cli/service/utils/WorkspaceManagerUtils.java

```diff
@@ -1,7 +1,7 @@
-package bio.terra.cli.utils;
```

Comment:
Side note: gut feeling, I think we're eventually going to want to rearrange these static methods once we start unit testing more of the codebase. Static methods are notoriously tricky to deal with in those scenarios.

That said, I wouldn't want to prematurely optimize anything yet — especially since this codebase doesn't use our usual Spring dependency injection approach, so my usual aesthetic won't necessarily apply here.

Reply:
Actually, I was hoping this would make testing a little easier since we can unit test this class separately -- i.e. test communication with WSM service separately from managing workspace context + communication with WSM service.

Originally I had all these methods as helpers in the WorkspaceManager class, but thought it was slightly more readable to have the WSM service communication separate from the context manipulation. Not sure if this will hold up as we add more to the codebase.

Reply:
I think we could make these non-static while still keeping this distinct from the manager class. I'd also tend towards wrapping data and behavior in class instances instead of static methods. It seems like `ApiClient` is the implicit argument always, but do we even need to expose it?

```
/** Calls Workspace Manager endpoints. */
public class WorkspaceManagerService {
  private final ApiClient;

  public WorkspaceManagerService(TerraUser terraUser, ServerSpecification server);
 
  public SystemVersion getVersion();
  WorkspaceDescription createWorkspace();
  ....
}
```

Reply:
Okay, I don't have strong feelings about static/not. I mostly want to separate out the code that communicates with Terra service from the code that manipulates the CLI context. I'll refactor these service utils classes to not have static methods.